src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => (
+
+  <div className="Person">
+    <h2 className="Person__name">
+      My name is
+      {person.name}
+    </h2>
+    <div className="Person__age">
+      {person.age > 0 && (
+        `i am ${person.age}`
+      )}
+    </div>
+    <p className="Person__partner">
+      {person.isMarried === true ? (
+        `${person.partnerName} is my
+        ${
+          person.sex === 'm' ? (
+            'wife'
+          ) : (
+            'husband'
+          )
+        }`
```

Comment:
Avoid ternary nesting, move this expression to variable

Reply:
still not fixed
<img width="736" alt="image" src="https://github.com/mate-academy/react_person/assets/70220667/e61a5023-8178-4204-9b42-1d04af2cd5c8">